desk/app/groups.hoon

```diff
@@ -431,14 +431,19 @@
   ?+    pole  [~ ~]
       [%x %gangs ~]  ``gangs+!>(xeno)
       [%x %init ~]  ``noun+!>([groups-light xeno])
-      [%x %init %v0 ~]  ``noun+!>([groups-light-ui xeno])
+      [%x %init %v0 ~]  ``noun+!>([groups-light-ui-0 xeno])
+      [%x %init %v1 ~]  ``noun+!>([groups-light-ui xeno])
       [%x %groups %light ~]  ``groups+!>(groups-light)
-      [%x %groups %light %v0 ~]  ``groups-ui+!>(groups-light-ui)
+      [%x %groups %light %v0 ~]  ``groups-ui-0+!>(groups-light-ui-0)
+      [%x %groups %light %v1 ~]  ``groups-ui+!>(groups-light-ui)
   ::
       [%x %groups ~]
     ``groups+!>(`groups:g`(~(run by groups) tail))
   ::
       [%x %groups %v0 ~]
+    ``groups-ui-0+!>(`groups-ui:zer:old:g`(~(run by groups) to-group-ui-0))
```

Comment:
NB: very very minor nitpick but I don't like that the mark is `foo-0` while the scry is `/foo/v0/`